ansible_collections/arista/avd/roles/eos_designs/templates/core_interfaces/ethernet-interfaces.j2

```diff
@@ -32,7 +32,18 @@ ethernet_interfaces:
 {%         endif %}
 {%         if switch.underlay_routing_protocol in ["ospf", "ospf-ldp"] %}
     ospf_network_point_to_point: true
-    ospf_area: {{ underlay_ospf_area }}
+{%             if p2p_interface.ospf_area is arista.avd.defined  %}
+    ospf_area: {{  p2p_interface.ospf_area }}
+{%             endif %}
+{%             if p2p_interface.ospf_authentication is arista.avd.defined %}
+    ospf_authentication: {{ p2p_interface.ospf_authentication | arista.avd.default(ospf_authentication) }}
+{%                 if p2p_interface.ospf_authentication == 'simple' %}
+    ospf_authentication_key: {{ p2p_interface.ospf_authentication_key | arista.avd.default(ospf_authentication_key) }}
+{%                 endif %}
+{%                 if  p2p_interface.ospf_authentication == 'message-digest' %}
```

Comment:
all these already implment the default fallback in the logic template. Please clean this one up, to only consume from the `core_interfaces_data` data model as applicable